wagtail/admin/templates/wagtailadmin/pages/listing/_page_title_cell.html

```diff
@@ -1,4 +1,4 @@
 {% load l10n %}
 <td id="page_{{ page.pk|unlocalize }}_title" class="title" data-listing-page-title>
```

Comment:
The `page` here should be updated to `instance`, as `page` is not in the cell's context.

And the column's `classname` should be here somehow...

```suggestion
<td id="page_{{ instance.pk|unlocalize }}_title" class="title {% if column.classname %}{{ column.classname }}{% endif %}" data-listing-page-title>
```

(The `if` is probably not needed)

Reply:
Oops, well spotted! Fixed in 113668cb4261d442604afbdf8c11008fcaca03c3. Have set it up so that `title` is the default classname, and if you override it on the column definition, it needs to be specified as `classname="align-top title"` - that way it's consistent with how we deal with the `listing` classname on the table.

Reply:
Nice, that makes more sense!